sql/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -377,18 +383,28 @@ private void authenticate(Channel channel, Properties properties) {
             );
             Messages.sendAuthenticationError(channel, errorMessage);
         } else {
-            try {
-                User user = authMethod.authenticate(userName, connProperties);
-                if (user != null && LOGGER.isTraceEnabled()) {
-                    LOGGER.trace("Authentication succeeded user \"{}\" and method \"{}\".",
-                        user.name(), authMethod.name());
-                }
-                session = sqlOperations.createSession(properties.getProperty("database"), user);
-                Messages.sendAuthenticationOK(channel)
-                    .addListener(f -> sendParamsAndRdyForQuery(channel));
-            } catch (Exception e) {
-                Messages.sendAuthenticationError(channel, e.getMessage());
+            authContext = new AuthenticationContext(authMethod, connProperties, userName, LOGGER);
+            if ("password".equals(authMethod.name())) {
```

Comment:
Hardcoded string should be replace with a static reference.

Reply:
the postgres module has no dependency on the enterprise users module which hold the AuthenticationMethodPassword class.
the string can still be made static though